README.md

```diff
@@ -25,6 +25,20 @@ Scalars, vectors, and higher-grade entities can be mixed freely and consistently
 ![Visual explanation of blades](https://raw.githubusercontent.com/pygae/clifford/master/docs/_static/blades.png)
 
 
+Quick Installation
+------------
+Requires Python version >=3.7
```

Comment:
I think we support 3.5 still?

Reply:
I was following what @hugohadfield wrote here: https://slides.com/hugohadfield/installing-python-and-clifford#/1 , but maybe that was just the latest python at the time.

Reply:
Ah yeah, I think those slides were for an old seminar we did and I probably just added the latest version of python at the time... Our CI runs (and often passes) on 3.5+ :)

Reply:
Also, thanks for the PR Ted! With the change to `version >=3.5` we should be OK to merge, what do you think @eric-wieser ?